docs/getting-started.md

```diff
@@ -276,21 +263,21 @@ RequestForAttestation {
 }
 ```
 
-#### 5.1.2. Sending an Attestation
+#### 5.2. Sending an Attestation
 
-Before we can send the request for an attestation to an Attester, we should first [create an Attester identity like above](#how-to-generate-an-identity).
+Before we can send the request for an attestation to an Attester, we should first create an Attester identity, like done [above](#how-to-generate-an-identity).
 
 ```typescript
 const attesterMnemonic = Kilt.Identity.generateMnemonic()
-// mnemonic: prepare neither execute excuse return visit claim hill around riot valid humor
+// Alternatively, a well-known mnemonic is -> prepare neither execute excuse return visit claim hill around riot valid humor
 const attester = Kilt.Identity.buildFromMnemonic(mnemonic)
-// attester.address: 4oFi7fgq9Bzd1vwjZUSoYuATiqSvy5ZJFbScvQL2jAwJkWB3
+// Using the well-known mnemonic, attester.address = 4tEpuncfo6HYdkH8LKg4KJWYSB3mincgdX19VHivk9cxSz3F
 ```
 
 If the Attester doesn't live on the same machine, we need to send them a message with the request.
 KILT contains a simple messaging system and we describe it through the following example.
 
-First, we create the request for an attestation message in which the Claimer automatically encodes the message with the public key of the Attester:
+First, we create the request for an attestation message in which the Claimer automatically encrypts the message with the public key of the Attester:
```

Comment:
It does not make sense to encode a message with a public key, but I am not sure if `encrypts` is what goes here either.

Reply:
encrypt is definitely closer to what was meant here. 😄 I think`in` and `automatically` should also be removed.

```suggestion
First, we create the request for attestation message which the Claimer encrypts with the public key of the Attester:
```